edx_prefectutils/mysql.py

```diff
@@ -115,26 +120,49 @@ def _drop_temp_tables(table, connection):
         query = "CREATE TABLE {table} ({table_schema})".format(table=table + '_temp', table_schema=table_schema)
         connection.cursor().execute(query)
 
+    manual_assignments = ', '.join(
+        '{}={}'.format(column, expression) for column, expression in table_column_manual_assignments
+    )
+    column_manual_assignments = 'SET {}'.format(manual_assignments) if manual_assignments else None
+
     try:
         if row and overwrite and not overwrite_with_temp_table:
             query = "DELETE FROM {table} {record_filter}".format(table=table, record_filter=record_filter)
             logger.debug("Deleting existing data for {table}".format(table=table))
             connection.cursor().execute(query)
 
-        query = """
-            LOAD DATA FROM S3 PREFIX '{s3_url}'
-            INTO TABLE {table}
-            FIELDS TERMINATED BY '{delimiter}' OPTIONALLY ENCLOSED BY '{enclosed_by}'
-            ESCAPED BY '{escaped_by}'
-            IGNORE {ignore_lines} LINES
-        """.format(
-            s3_url=s3_url,
-            table=table if not overwrite_with_temp_table else table + '_temp',
-            delimiter=field_delimiter,
-            enclosed_by=enclosed_by,
-            escaped_by=escaped_by,
-            ignore_lines=ignore_num_lines,
-        )
+        if not column_manual_assignments:
```

Comment:
Even though I am conditionally adding `SET` parameter in query but test is still failing